global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "7.0.100-preview.7.22377.5",
+    "version": "7.0.100-preview.6.22352.1",
```

Comment:
Probably a result of merge? Should be preview 7. Probably easier just take global.json from main.

Reply:
Fixed it